docs/source/tutorials/00_api_basics.ipynb

```diff
@@ -1211,6 +1213,137 @@
     "For a more detailed illustration of pipeline building, please check [this](https://docs.lineapy.org/en/latest/guide/build_pipelines/index.html#) tutorial."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "080181c2-e5b0-47ff-90e7-a89b21573a99",
+   "metadata": {},
+   "source": [
+    "## Using helper functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf4d338b-a14a-40d0-83e8-da0994fa9bdc",
+   "metadata": {},
+   "source": [
+    "On top of the various functionalities introduced above, there are a few helper functions that we have made public. We think these functions can be very helpful when performing repeated operations or trying out variations on experiments. These functions are introduced below:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a264b9aa-3c45-4727-8a68-ad3ae28e58fb",
+   "metadata": {},
+   "source": [
+    "### Using get_function to perform the same operation on different input"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "060a8214-4fff-4ab8-8dbf-93c751dd15cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = 1\n",
+    "b = a**2\n",
+    "squareart = lineapy.save(b, \"squared\")\n",
+    "sq = lineapy.get_function([\"squared\"], input_parameters=[\"a\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "554a2ab4-552d-4c60-ab40-376eaef1de37",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "9"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sq(a=3)[\"squared\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "347f8a43-95fa-4275-9bb8-6671a3de6c08",
+   "metadata": {},
+   "source": [
+    "#### What's happening?\n",
+    "\n",
+    "Underneath the hood, get_function uses parametrized version of a pipeline with a single artifact and configurable inputs. In this case, since only one artifact \"squared\" was used to construct the pipeline, it returns a dict with just updated value. The interface however is that of a normal function that was imported and that returns a dict. For more reference on Pipelines and parametrized inputs, refer to our advanced tutorials on [parametrized pipelines](https://docs.lineapy.org/en/latest/guide/build_pipelines/pipeline_parametrization.html). To explore how else get_function can help you, refer to the [demo]() on using get_function for dev."
```

Comment:
One thing I want to add here(after explaining what's underneath the hood of the function) is users can test the function directly before it gets deployed to the pipeline.